### ISSUE TITLE
Start and end delimiters

### ISSUE THREAD

**Issue by paulehoffman** [timbray/blueskidgo#2]
>It would probably be better to use two different charactures to mark the start and end of the assertion.

This seems true (for parsing ease), so choose a different instrument for the end delimiter. Proposed: 🎹 (MUSICAL KEYBOARD, U+1F3B9)

**Comment by timbray** [comment]
Yes, good idea.  Should this project turn from a quickie prototype to something people want to use, I'll make that change.
